osu.Android/OsuGameActivity.cs

```diff
@@ -16,7 +16,10 @@
 namespace osu.Android
 {
     [Activity(Theme = "@android:style/Theme.NoTitleBar", MainLauncher = true, ScreenOrientation = ScreenOrientation.FullUser, SupportsPictureInPicture = false, ConfigurationChanges = ConfigChanges.Orientation | ConfigChanges.ScreenSize, HardwareAccelerated = false, LaunchMode = LaunchMode.SingleInstance)]
-    [IntentFilter(new[] { Intent.ActionDefault, Intent.ActionSend }, Categories = new[] { Intent.CategoryDefault }, DataPathPatterns = new[] { ".*\\.osz", ".*\\.osk" }, DataMimeType = "application/*")]
+    [IntentFilter(new[] { Intent.ActionView }, Categories = new[] { Intent.CategoryDefault }, DataScheme = "content", DataPathPattern = ".*\\\\.osz", DataHost = "*", DataMimeType = "*/*")]
+    [IntentFilter(new[] { Intent.ActionView }, Categories = new[] { Intent.CategoryDefault }, DataScheme = "content", DataPathPattern = ".*\\\\.osk", DataHost = "*", DataMimeType = "*/*")]
```

Comment:
Data MIME type `"*/*"` seems a bit too broad. Even if the file manager / browser fails to guess the correct MIME type of the file, it would set it to `application/octet-stream` (or even `application/*` for some obscure file managers), so `"application/*"` should fit in there for both of them instead of `"*/*"`

Reply:
Some file managers prompt users to select type of file (Image, Picture, Document, etc.) for unknown file extensions. A user might think: "osu! beatmap, that's like a song, right?"

Song would be MIME type `audio/*`. We want to present the user with the option of importing even if they select alternative options. We should leave `*/*` as is.